File: python/matrizes/ordem_crescente.py
print("---------------------ORDEM CRESCENTE----------------------\n")

list_numbers = []

for i in range(20):
    value = int(input(f"Digite o {i+1}º valor: "))
    list_numbers.append(value)

list_numbers.sort()

print("----------------------------------------------")
print("OS NÚMEROS DIGITADOS FORAM")

for element in list_numbers:
    print(element, end=" ")

File: java/matrizes/OrdemCrescente.java
package matrizes;

import java.util.Scanner;
import java.util.Arrays;

import libraries.TerminalFormating;

public class OrdemCrescente {
    
    public static void main(String[] args) {

        final int REGISTER_NUMBER = 20;

        int[] value = new int[REGISTER_NUMBER];

        Scanner input = new Scanner(System.in);
        TerminalFormating terminal = new TerminalFormating();

        //inicio do programa
        terminal.writeProgramTitle("ORDEM CRESCENTE");

        for(int i = 1; i <= REGISTER_NUMBER; i++) {
            System.out.print("\nDigite o " + i + "º valor: ");
            value[i-1] = input.nextInt();
        }
        
        Arrays.sort(value);

        System.out.println("-----------------------------------");
        for (int i = 0 ; i < REGISTER_NUMBER; i++) {
            System.out.print(value[i] + " ");
        }
    }
}
